scripts/src/main/resources/scripts/migrations/2023.06.001

```diff
@@ -47,3 +47,22 @@ find "${DEVON_IDE_HOME}"/workspaces/ -mindepth 2 -maxdepth 2 -name devon.propert
 do
   doAdaptConfigurationFiles "${foundfile}"
 done
+
+plugins_path="${DEVON_IDE_HOME}/plugins"
+
+if [ ! -d "${plugins_path}" ]
+then
+	mkdir "${plugins_path}"
+else
+	doWarning "Plugins directory already exists!"
+fi
+
+find "${DEVON_IDE_HOME}"/software -maxdepth 1 -type d -name "*-plugins" -print | while read found_folder
```

Comment:
Great that you added the migration. 👍 
As we only have 2 specific candidates here (`vscode-extensions` and `eclipse-plugins`) it does not make much sense trying to solve this in a generic way esp. as `vscode-extensions` does not match `*-plugins`.
I would simply hardcode these two folders here. In the end all we need is just
```
mv "${DEVON_IDE_HOME}/software/vscode-extensions" "${DEVON_IDE_HOME}/plugins/vscode"
mv "${DEVON_IDE_HOME}/software/intellij-plugins" "${DEVON_IDE_HOME}/plugins/intellij"
```

Android-studio has not yet been released and eclipse is not yet separated so KISS is key.

Reply:
Feel free to check the directories for existence to suppress `mv: cannot stat 'software/vscode-extensions': No such file or directory` error.

Reply:
I've applied KISS now and added checks for existing directories.